src/components/ViewComments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getComments } from '../utils/api';
+
+const ViewComments = () => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getComments(article_id).then((commentsFromApi) => {
+      setComments(commentsFromApi);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  if (isLoading) return <p>Loading...</p>;
+  return (
```

Comment:
this looks good, consider what happens if comments.length in 0 - this might make sense once you get to adding comments. If there are none we could display a 'be first to comment!' message or similar